src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/model/licence/additionalConditions/ElectronicMonitoringAdditionalCondition.kt

```diff
@@ -0,0 +1,43 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.publicApi.model.licence.additionalConditions
```

Comment:
<kbd><img width="1229" alt="Screenshot 2023-10-27 at 11 46 27" src="https://github.com/ministryofjustice/create-and-vary-a-licence-api/assets/1517745/eb18c0a6-35a4-4670-88da-51de423ac4c9"></kbd>